rosbag2_transport/test/rosbag2_transport/test_play.cpp

```diff
@@ -66,7 +66,7 @@ TEST_F(RosBag2PlayTestFixture, recorded_messages_are_played_for_all_topics)
 
   auto prepared_mock_reader = std::make_unique<MockSequentialReader>();
   prepared_mock_reader->prepare(messages, topic_types);
-  reader_ = std::make_unique<rosbag2_cpp::Reader>(std::move(prepared_mock_reader));
+  auto reader = std::make_unique<rosbag2_cpp::Reader>(std::move(prepared_mock_reader));
```

Comment:
Is there an associated change on the test fixture that removes the `reader_` member?

Reply:
I've tabled that for a follow up PR. My plan is to re-write the test suite pretty much completely. There's a lot of functionality in there which was written at the time because we've lacked these in `rcl*`. Most of them are now available and my goal with that PR would be to harden the tests altogether.